tests/e2e/reports/report-date-filter.wdio-spec.js

```diff
@@ -99,9 +99,9 @@ describe('Filters reports', () => {
     await reportsTab.filterByDate(moment('05/16/2016','MM/DD/YYYY'), moment('05/17/2016','MM/DD/YYYY'));
     await (await reportsTab.firstReport()).waitForDisplayed();
 
-    expect(await reportsTab.allReports().length).to.equal(2);
-    expect(await reportsTab.reportsByUUID(savedUuids[1]).length).to.equal(1);
-    expect(await reportsTab.reportsByUUID(savedUuids[3]).length).to.equal(1);
+    expect(await (await reportsTab.allReports()).length).to.equal(2);
```

Comment:
I was expecting to be able to remove one of the two `await` like so
```ts
expect((await reportsTab.allReports()).length).to.equal(2);
```

But I've tried reverting this change and it's still returning `2`. Do we really need to include these changes?

Reply:
These were failing for me locally, so I think this is a race condition that sometimes works.

Reply:
Good job catching it!